Porting FxCop Rules to Roslyn.md

```diff
@@ -0,0 +1,66 @@
+# Porting Managed Code Analysis Rules to Roslyn
+
+Visual Studio 2015 shipped with over 300 Code Analysis rules for managed code. These rules were all written as binary analysis rules using the FxCop engine. We are now engaged in an effort to rewrite some of these rules as Roslyn analyzers. This has the following benefits:
+
+* You get live analysis as you type in VS.
+* Roslyn analyzers can be accompanied by fixers.
+
+However, we do not envision the new Roslyn-based managed analysis rules as a strict port of the FxCop rules, for various reasons:
+
+* FxCop originally started as a "Framework Cop". It was intended to enforce guidelines for .NET Framework API design and BCL (Basic Class Library) usage. But the name "FxCop" was conflated with the particular binary analysis technology with which it was implemented, and so it came to refer to all the different kinds of rules that ship in VS. So FxCop means different things to different people.
```

Comment:
I don't see what this text has to do with the price of apples. :) Also, the history is not quite right. FxCop started with the .NET design guidelines, we then integrated new checks based on common BCL API misuse patterns observed by the CLR experts (including test). We then sketched out a broader analysis space, for internationalization, power consumption checks for mobile, 32/64 bit compatibility, etc.). When FxCop was integrated into VS, further investment in analysis essentially halted. Getting to your point here, why we're not working with a strict port, what I'd focus on is that FxCop aggregates a number of quality concerns (API surface area, core BCL usage correctness, internatlionalization, perf, etc.) that argue for an unbundling, which is driving the refactoring.